third_party/kythe/README.google

```diff
@@ -1,5 +1,5 @@
-URL: https://github.com/google/kythe/tree/250bb910b0edb5ccf99971692f8068a66ba87eac
-Version: 250bb910b0edb5ccf99971692f8068a66ba87eac
+URL: https://github.com/google/kythe/tree/c9f42bfb7e5b8d33834232e181fcf6a4e8105629
```

Comment:
Yes, I went ahead and reran upgrade_kythe as part of building the new image. This will update hte commit number that we are using. Looks like there were no changes to this part of the tree otherwise, but at leas it's the same version as the image.